WTG.Analyzers.Test/TestData/CodeContractsAnalyzer/Assert/Result.cs

```diff
@@ -0,0 +1,10 @@
+using System.Diagnostics.Contracts;
+
+public static class Bob
+{
+	public static object Method()
+	{
+		var value = new object();
+		return value;
```

Comment:
Is it worth replacing `Contract.Assert` with `Debug.Assert`? I know we rarely use it, but I do recall that I have an `Assert` somewhere for safety and don't remember where. :smile:

Reply:
I looked into that, but decided it caused too many issues (particularly in unit tests where it stops to ask the user what they want to do.)